core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -171,8 +167,8 @@ private object RTS {
    */
   final class FiberContext[E, A](rts: RTS, val fiberId: FiberId, val unhandled: List[Throwable] => IO[Nothing, Unit])
```

Comment:
The type of `unhandled` needs to change from `List[Throwable] => IO[Nothing, Unit]` to `Cause[Nothing] => IO[Nothing, Unit]`.

Reply:
This should simplify code since usually `cause.toThrowable :: Nil` is being passed to `unhandled`.